bigquery/deploy/README.md

```diff
@@ -1,124 +1,157 @@
-    * Change `Machine type` to `4 vCPUs`. (Otherwise will get Insufficient CPU error.)  
-    If you need more memory for Elasticsearch, you may need `n1-highmem-4`. [Elasticsearch recommends](https://www.elastic.co/guide/en/elasticsearch/reference/current/heap-size.html) the VM has at least
-    twice the memory you are using for Elasticsearch.
-    * Click `Advanced edit` and under `Service account`, select the indexer service account you just created. Click `Save`.
-      * If you are using the default Compute service account instead of indexer,
-        click `Allow full access to all Cloud APIs`. (This is automatically
-        selected for custom service accounts.)
-    * Click `Create`.
-  * After cluster has finished creating, run this command to point `kubectl` to
-  the right cluster.
-    ```
-    gcloud config set project EXPLORER_PROJECT
-    gcloud container clusters get-credentials elasticsearch-cluster --zone MY_ZONE
-    ```
-    `MY_ZONE` is the [zone where elasticsearch-cluster is running](https://console.cloud.google.com/kubernetes/list),
-    e.g. `us-central1-a`.
-
-* Run Elasticsearch on GKE
-  * Deploy Elasticsearch. From project root:
-    ```
-    kubernetes-elasticsearch-cluster/deploy.sh MY_DATASET
-    ```
-    Note: This will delete all existing data in the index; re-deploy 
-    with caution.
-  * Test that Elasticsearch is up. ES_CLIENT_POD is something like
-  `es-client-595585f9d4-7jw9v`.
-    ```
-    kubectl get pods
-    kubectl exec -it ES_CLIENT_POD -- /bin/bash
-    curl localhost:9200
-    ```
-
-* Update and run indexer on GKE
-  * If you did the "Run Elasticsearch on GKE" step a while ago, run
-    these commands to point `gcloud` and `kubectl` to the right project.
-    ```
-    # See what projects gcloud and kubectl are configured for
-    gcloud config get-value project
-    kubectl config current-context
-    # Point gcloud and kubetl to the right project
-    gcloud config set project MY_PROJECT
-    gcloud container clusters get-credentials elasticsearch-cluster --zone MY_ZONE
-    ```
-    `MY_ZONE` is the [zone where elasticsearch-cluster is running](https://console.cloud.google.com/kubernetes/list),
-    e.g. `us-central1-a`.
-  * We recommend you delete the index, to start from a clean slate.
-    ```
-    kubectl get pods
-    kubectl exec -it ES_CLIENT_POD -- /bin/bash
-    curl -XDELETE localhost:9200/<MY_DATASET>
-    curl -XDELETE localhost:9200/<MY_DATASET>_fields
-    ```
-  * Make sure the files in `dataset_config/MY_DATASET` are filled out.
-    * Make sure `deploy.json` and the `authorization_domain` field in
-      `dataset.json` are filled out.
-  * From project root, run `bigquery/deploy/deploy-indexer.sh MY_DATASET`, where
-  MY_DATASET is the name of the config directory in `dataset_config`.
-  * Verify the indexer was successful:
-    ```
-    kubectl get pods
-    kubectl exec -it ES_CLIENT_POD -- /bin/bash
-    curl localhost:9200/_cat/indices?v
-    ```
+# Deploying on GKE
```

Comment:
I'm not able to deploy 1000 Genomes. My deploy.json only has project_id in it. Are you able to deploy 1000 Genomes?

I deleted my cluster, ran create-cluster.sh, then ran deploy.sh.

After `kubernetes-elasticsearch-cluster/deploy.sh`, es-master status is "Unschedulable and 1 more issue".

![master](https://user-images.githubusercontent.com/10929390/51566739-42e18580-1e4a-11e9-85a6-d726e7cb1906.png)

Reply:
Hmm...I've run it three times more deleting the cluster and re-deploying from scratch and it worked (with no changes to `deploy.json` besides project_id).

Would you mind trying again and maybe wait a minute after creating the cluster, before deploying (making sure it has the green check mark). It's possible the cluster master upgrade scenario I mentioned offline happened to you, especially since I see that three other clusters in the project were missing data (probably deleting), so GCP may have been rebalancing or something.